package.json

```diff
@@ -5,6 +5,8 @@
   "license": "UNLICENSED",
   "scripts": {
     "android": "react-native run-android",
+    "android-release": "yarn android --mode release",
+    "android:bundle:release": "npx react-native build-android --mode=release",
```

Comment:
script to generate the app bundle file which we need to upload to Google Play Console to distribute to testers

the bundle file is named `app-release.aab` and will be placed on `/freighter-mobile/android/app/build/outputs/bundle/release/app-release.aab`